src/mediumComplexityProblem1/IncomeTaxCalculation.java

```diff
@@ -0,0 +1,27 @@
+package mediumComplexityProblem1;
+
+import java.util.Scanner;
+
+public class IncomeTaxCalculation {
+
+	public static void main(String[] args) {
+		double tax = 0;
+		double it;
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter income ");
+		it = sc.nextDouble();
```

Comment:
Add entire logic under function and call that function from main method

Reply:
Done